crawler/twitter.js

```diff
@@ -27,59 +45,91 @@ exports.fetchTweets = function(query, maxID, i) {
 
         if (typeof data.statuses !== "object" || data.statuses.length < 1) {
             console.log("No data!");
-            return;
+            return i;
         }
 
-        const oldestStatus = data.statuses[data.statuses.length-1];
-        const oldestID_str = oldestStatus.id_str;
-        const oldestID = parseInt(oldestID_str);
+        for (var count = 0; count < data.statuses.length; count++) {
 
-        console.log(maxID, oldestID_str, oldestID, oldestStatus.created_at, '\n');
-
-        /*
-        data.statuses[0].created_at.split(' ').splice(1, 2).join(' ');
-        var split = data.statuses[0].created_at.split(' ');
-        var year = split[split.length - 1];
-        var date_calc = split[split.length - 4];
-        //console.log(date_calc);
-        //console.log(year, data.search_metadata.count);
-
-        try {
-            var_max_id = data.statuses[99].id;
-        } catch (err) {}
-        //console.log(" printing id :"+data.statuses[i].id);
-
-        for (var count = 0; count < data.search_metadata.count; count++) {
-            console.log(data.statuses[count].lang,
-                data.statuses[count].created_at,
-                data.statuses[count].id,
-                data.statuses[count].text,
-                data.statuses[count].retweet_count,
-                data.statuses[count].favorite_count);
+            retext().use(sentiment).use(function() {
```

Comment:
You should move the analysis part in a new function. You repeat the same code in line 103.